com.microsoft.mrtk.input/Tests/Runtime/Utilities/TestHand.cs

```diff
@@ -80,6 +80,18 @@ public override IEnumerator RotateTo(Quaternion newRotation, int numSteps = Inpu
             }
         }
 
+        /// <inheritdoc />
+        public override IEnumerator AimAt(Vector3 target, int numSteps = InputTestUtilities.ControllerMoveStepsSentinelValue, bool waitForFixedUpdate = true)
+        {
+            InputTestUtilities.SetHandAnchorPoint(handedness, Simulation.ControllerAnchorPoint.Device);
```

Comment:
Aiming is not possible without the Device anchor point, which is a bummer. I contemplated moving the anchor point back to the previous anchor point after performing the aim-at operation, but I thought that would actually confuse things a lot more because any subsequent commands to the hand would then snap it out of alignment.